vimeo-networking/src/main/java/com/vimeo/networking/utils/VimeoNetworkUtil.java

```diff
@@ -71,7 +72,8 @@ public static Gson getGson() {
      */
     public static GsonBuilder getGsonBuilder() {
         // Example date: "2015-05-21T14:24:03+00:00"
-        return new GsonBuilder().setFieldNamingPolicy(FieldNamingPolicy.LOWER_CASE_WITH_UNDERSCORES)
+        return new GsonBuilder().registerTypeAdapterFactory(new Stag.Factory())
+                .setFieldNamingPolicy(FieldNamingPolicy.LOWER_CASE_WITH_UNDERSCORES)
```

Comment:
If we're annotating every single model then we shouldn't need this naming policy anymore